terraform/ami-datasource.tf

```diff
@@ -0,0 +1,18 @@
+data "aws_ami" "ubuntu" {
+  most_recent = true
+
+  filter {
+    name   = "name"
+    values = ["ubuntu/images/hvm-ssd/ubuntu-jammy-22.04-amd64-server-*"]
+  }
+
+  filter {
+    name   = "virtualization-type"
+    values = ["hvm"]
+  }
+
+  owners = ["099720109477"] # Canonical
```

Comment:
I didn't understand the need for this, can you please tell me what the owner means here

Reply:
In this case this filter will only find images owned by the account id of 099720109477